### DISCUSSION TITLE
Test Branch

### DISCUSSION THREAD

**Discussion by Mirrorgo** [DemocracyDevelopers/AuditVisualizer#49] [Git Management]
We’ve set up various test branches and corresponding deployment sites for easier testing and peer reviews. Below is a list of the available test branches along with the URLs of their deployed versions. Feel free to use these for your testing and review processes.

#### 📝 **Test Branches and Deployment URLs**:

| **Test Branch**      | **Deployment URL**                             | **Owner**          |
|----------------------|------------------------------------------------|--------------------|
| `test/lwz`           | https://audit-visualizer-lwz.vercel.app/ | @Mirrorgo  |
| `test/linsheng_test`  |                                                | @Linkgls            |
| `test/yilin_test`     |   https://audit-visualizer-9c3m.vercel.app/                 | @lvyl9909 |
| `test/emily-test`     |   https://audit-visualizer-mu.vercel.app/    | @ylimezhang               |
| `test/weijia-test`     | https://audit-visualizer-psi.vercel.app/ | @Auroraphile  |

#### 🔧 **How to Use**:
1. Visit the deployment URL for each test branch to check the feature in its testing environment.
2. Conduct your tests and review the changes directly on the site.
3. Please provide feedback by leaving comments on the corresponding pull requests (PRs) or via our internal review system.

#### ✅ **Review Guidelines**:
- Ensure that all functional and visual aspects of the feature work as expected.
- Cross-check for responsiveness, accessibility, and overall user experience.
- Report any bugs, issues, or suggestions in the dedicated channel or via PR comments.

Thank you for your collaboration and contribution to our testing process!

**Comment by lvyl9909** [comment]
For tutorial pages: https://audit-visualizer-i7vp.vercel.app/tutorial

**Comment by Auroraphile** [comment]
For dashboard: 
      url:https://audit-visualizer-psi.vercel.app/upload
      branch: test/weijia-test
